### PR TITLE
updated css

### PR DESCRIPTION
**Pull request by codiebeulaine**
@miltontony ready for review

### CODE REVIEW

**Comment by codiebeulaine** [comment]
Spacing on free text questions
Menu color static black with white background
Button color inherits from accent1 and block background color
Styled search results

File: freebasics/static/css/template-base.css [hunk 1]
```diff
@@ -309,4 +308,9 @@ h6{
 .button-div a, .button-div input[type="submit"]{
   width: 12em;
   margin: 0 auto;
+  display: block;
+}
+.free-text-form input[type="text"], input[type="number"]{
```
Comment (miltontony) [inline freebasics/static/css/template-base.css]: the second part applies to all `input[type="number"]`. You should prefix it with `.free-text-form input[type="number"]`

File: freebasics/templates/search/search_results.html [hunk 2]
```diff
@@ -5,30 +5,21 @@
 {% if search_results %}
 <div class="block-search fb-body fb-card">
   <h6 class="block-heading fb-accent-1">Search Results</h6>
-  <h6 class="sub-heading">{{search_query}}</h6>
+  <h6 class="sub-heading">Search Results for {{search_query}}</h6>
```
Comment (miltontony) [inline freebasics/templates/search/search_results.html]: You should put these words in `{% trans "Search results for" %}` (same for seach results above)

**Comment by miltontony** [comment]
small minor stuff to fix, otherwise :+1:

**Comment by codiebeulaine** [comment]
@miltontony fixed, ready for review again

File: freebasics/templates/search/search_results.html [hunk 3]
```diff
@@ -5,30 +5,21 @@
 {% if search_results %}
 <div class="block-search fb-body fb-card">
   <h6 class="block-heading fb-accent-1">Search Results</h6>
```
Comment (miltontony) [inline freebasics/templates/search/search_results.html:7]: I was referring to this one too

Comment (miltontony) [inline freebasics/templates/search/search_results.html:7]: Can you just double check that strings in all these templates are in trans blocks

**Pull request merged**